tests/unit/cluster/hostnames.tcl

```diff
@@ -174,12 +174,12 @@ test "Verify the nodes configured with prefer hostname only show hostname for ne
 
     # Also make sure we know about the isolated master, we 
     # just can't reach it.
+    resume_process [srv 0 pid]
     set master_id [R 0 CLUSTER MYID]
     assert_match "*$master_id*" [R 6 CLUSTER NODES]
 
     # Stop dropping cluster packets, and make sure everything
     # stabilizes
-    R 0 DEBUG DROP-CLUSTER-PACKET-FILTER -1
     R 6 DEBUG DROP-CLUSTER-PACKET-FILTER -1
```

Comment:
We shoudn't unpause node 0 before this check: "Also make sure we know about the isolated master, we just can't reach it."

You unpaused it here just to get MYID of node 0, right? We can fetch `R 0 CLUSTER MYID` early in the test case, before we pause node 0, instead.

Reply:
Sure